Slim/Formats/FLAC.pm

```diff
@@ -57,6 +57,7 @@ my %tagMapping = (
 	'DESCRIPTION'               => 'COMMENT',
 	'ORIGINALYEAR'              => 'YEAR',
 	'UNSYNCEDLYRICS'            => "LYRICS",
+	'WORK'			    => 'WORK',
```

Comment:
Are there other file formats which would provide an equivalent tag?

And please format using spaces only between the first `'WORK'` and the equal sign 😁.

Reply:
Uh... oh... and then there's musicbrainz...

https://picard-docs.musicbrainz.org/en/appendices/tag_mapping.html#musicbrainz-work-id
https://picard-docs.musicbrainz.org/en/appendices/tag_mapping.html#work-title

Reply:
Yes, definitely need to consider other file formats, just did flac for now so I'd have something to play with for testing.

Reply:
So I just downloaded Peter Gabriel's latest album "i/o". It's actually a "disc set" with the Dark Side Mix on one disc, the Bright Side Mix on the other. I thought... I could use these mixes as works? Not exactly classical music, but it would give me a way to group those independently whenever I wanted. So I tagged them as two works in my preferred tag editor (Meta for Mac). Alas... that wrote the data to a tag called `GROUPING`. Turns out that's [yet something different](https://picard-docs.musicbrainz.org/en/appendices/tag_mapping.html#grouping-3).

So I started some more googling, wondering, whether I should ask you to treat `GROUPING` the same as `WORK`. And found discussions around exactly this topic, as [Apple seems to have confused the two, too](https://discussions.apple.com/thread/8429166) (at least at some point).

What's your thought on this? I mean... even if it's not the same thing, they could be treated the same, as both are essentially a way to group tracks from an album into some other entity, aren't they?

BTW: Meta would call the `WORK` tag "Workname"...

Reply:
Moving on to tagging changes for all formats, now. Nothing is ever simple, is it?

I'll have a look at this GROUPING vs WORK stuff.

I think, after this, we're ready for a code review and experimental release. I'd at least like to get a few people using it to make sure I haven't broken anything. You wouldn't need a classical collection or WORK tags to help in that regard.

Reply:
@michaelherger OK, on GROUPING, for couple of classical releases I ran through Picard (with classical music plugins added), GROUPING and WORK were both created with similar (but not always identical) data - on one album, WORK included the year of composition in brackets whereas GROUPING was just the plain work name. But yes, they seem essentially the same, subject to the vagaries of Musicbrainz.

I'll code it to map WORK or GROUPING to works.title in the database, with WORK taking precedence.

I've now tested mp3, m4a and ogg as well as flac, and it's all working well. Look out for updates very soon...